bluesky/tests/test_est_time.py

```diff
@@ -0,0 +1,105 @@
+from bluesky.simulators import est_time
+from ophyd.sim import hw
+from bluesky.run_engine import RunEngine
+from ophyd.telemetry import TelemetryUI
+from bluesky.plans import scan, count
+from bluesky.plan_stubs import mv, sleep
+import pytest
+
+#setup some testing objects
+RE=RunEngine()
+hw = hw()
+motor1 = hw.motor1
+motor2 = hw.motor2
+det1 = hw.det1
+det2 = hw.det2
+
+###Start by testing that the telemetry recording is working
+
+#test telemetry recording.
+def test_telemetry_recording():
+    length = len(TelemetryUI.telemetry) #find the current length of the telemetry dictionary
+    RE(mv(motor1, 5)) #move motor1
+    assert len(TelemetryUI.telemetry) == length + 1 #check an entry has been added to the dictionary
+
+    length = len(TelemetryUI.telemetry) #find the current length of the telemetry dictionary
+    RE(count([det1])) #stage, trigger and unstage det 1
+    assert len(TelemetryUI.telemetry) == length + 1 #check an entry has been added to the dictionary 
+                                                   #(3 once stage and unstage are added)
+
+
+#set up the telemetry dictionary with some known values. 
+def setup_telemetry():
```

Comment:
This is a good place to use a pytest fixture.

Reply:
I will look into pytest fixtures.